squad-server/plugins/basic_stats.js

```diff
@@ -0,0 +1,738 @@
+/* eslint-disable no-duplicate-case */
```

Comment:
Need a reasoning on why these.

Reply:
Looking at the code, I think this can be removed without issue. This flag prevents multiple uses of `case` with the same value which has not been done.